scripts/release-notes.py

```diff
@@ -171,11 +171,11 @@ def main(token, repo, branch, num_commits, exclude_dependabot, verbose):
         print()
         print('### 📊 UI Changes')
         print()
-        main(token, 'jaeger-ui', 'main', None, exclude_dependabot, False)
+        main(token, 'jaeger-ui', 'main', None, exclude_dependabot, verbose)
 
     # Print pull requests in the 'UNCATTEGORIZED' category
     if other_results:
-        print(f'### 💩💩💩 The following commits cannot be categorized (missing changeglog labels):\n')
+        print(f'### 💩💩💩 The following commits cannot be categorized (missing changelog labels):\n')
```

Comment:
```suggestion
        print(f'### 💩💩💩 The following commits cannot be categorized (missing changelog labels):\n')
        print(f'### 💩💩💩 The following commits cannot be categorized (missing "changelog:*" labels):\n')
```
and after the loop let's add this
```
        print(f'### 💩💩💩 Please attach labels to these ^^^ PRs and rerun the script.\n')
        print(f'### 💩💩💩 Do not include this section from the changelog.\n')
```